src/routeObject.js

```diff
@@ -1,6 +1,51 @@
 import React from 'react';
 import {Route, IndexRoute} from 'react-router';
+
 import App from './pages/App';
+import NotFound from './pages/PageNotFound';
+import store from './store';
+
+const validPathTest = (course, lesson, path) => {
+  const lessons = store.getState().lessons;
+  const courses = store.getState().context['courseContext'].keys();
+  let key;
+
+  if(!lesson){
```

Comment:
Flisespikking, men jeg foretrekker `if (lesson) { ... } else { ... }` i stedet for  `if (!lesson) { ... } else { ... }`